### PR TITLE
expose more executables

### PR DESCRIPTION
**Pull request by jmtcsngr**
expand list of executables exposed with:
  plot-ampliconstats
  plot-bamstats

### CODE REVIEW

File: docker/Dockerfile
Comment (kjsanger) [inline docker/Dockerfile:111]: Don't these require gnuplot to work? I can't see that being installed in the Dockerfile

**Review by kjsanger — CHANGES_REQUESTED** [review]

File: docker/Dockerfile
Comment (jmtcsngr) [reply docker/Dockerfile:112]: Indeed it is. Thanks!

For my future reference, `plot-ampliconstats  --help` does not complain about the missing _Gnuplot_.

_Gnuplot_ should be included in next commit. Would like to manually squash commits, so it is atomic change. @kjsanger if you are happy with the next commit, please give me a chance to rebase before merging.

**Comment by jmtcsngr** [comment]
Rebased and squashed. So ready to merge if you are happy @kjsanger.

**Review by kjsanger — APPROVED** [review]

**Pull request merged**
